src/main/java/gyro/aws/apigatewayv2/AuthorizerResource.java

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2020, Brightspot.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.aws.apigatewayv2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import gyro.aws.AwsResource;
+import gyro.aws.Copyable;
+import gyro.aws.iam.RoleResource;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Id;
+import gyro.core.resource.Output;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Required;
+import software.amazon.awssdk.services.apigatewayv2.ApiGatewayV2Client;
+import software.amazon.awssdk.services.apigatewayv2.model.Authorizer;
+import software.amazon.awssdk.services.apigatewayv2.model.AuthorizerType;
+import software.amazon.awssdk.services.apigatewayv2.model.CreateAuthorizerResponse;
+import software.amazon.awssdk.services.apigatewayv2.model.GetAuthorizersResponse;
+
+/**
+ * Create an authorizer.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *     aws::authorizer example-authorizer
+ *         api: $(aws::api example-api)
+ *         name: "example-authorizer"
+ *         authorizer-type: JWT
+ *
+ *         identity-sources: [
+ *             '$request.header.Authorization'
+ *         ]
+ *
+ *         jwt-configuration
+ *             audiences: [
+ *                 "example-audience"
+ *             ]
+ *
+ *             issuer: "https://cognito-idp.us-east-2.amazonaws.com/us-east-2_pQJThFAx4"
+ *         end
+ *     end
+ */
+@Type("authorizer")
```

Comment:
To make it less generic 
```suggestion
@Type("api-gateway-authorizer")
```